xml_struct/Cargo.toml

```diff
@@ -6,6 +6,7 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+anyhow = "1.0.69"
```

Comment:
This is not the latest version of `anyhow`, but it's the one that's currently specified in comm-central's `Cargo.lock`.